action_server/src/sema4ai/action_server/_new_project.py

```diff
@@ -1,19 +1,38 @@
-import io
 import logging
 import os
-import sys
-import zipfile
-from urllib.request import urlopen
-
-TEMPLATE_URL = (
-    "https://github.com/robocorp/template-action/archive/refs/heads/master.zip"
-)
 
 log = logging.getLogger(__name__)
 
 
-def create_new_project(directory: str = "."):
+def create_new_project(directory: str = ".", template_name: str = "") -> None:
```

Comment:
I was actually taking a look in the code and this should be changed to return 0 or an error code (1 is fine) and the `_main_retcode` should return the error code in that case too.

Note that it's fine to return 0 if it was cancelled with a `Ctrl+0`, but if an exception happens we should print it and return a non-0 value.

Reply:
@fabioz done!